.rubocop.yml

```diff
@@ -84,6 +70,9 @@ Performance/Casecmp:
 Security/YAMLLoad:
   Enabled: false
 
+Style/AccessorGrouping:
+  Enabled: false
```

Comment:
This one caused violations in this repo so I turned it off. Enabling it would mean we would lose our ability to add comments for each individual accessor:

```ruby
attr_accessor :one # Comment describing :one
attr_accessor :two # Comment describing :two
```

Rubocop wanted:

```ruby
attr_accessor :one, :two
```